book/intro.qmd

```diff
@@ -7,57 +7,57 @@
 |                | of land use in urban environment                 |
 +----------------+--------------------------------------------------+
 | **Output**     | Prediction of the quality of life indicators     |
-|                | following a modelled scenarios of development    |
+|                | following modelled scenarios of development      |
 |                |                                                  |
-|                | study case: Tyne and Wear County (Local          |
-|                | Authorities: Gateshead, Newcastle Upon Tyne,     |
-|                | North Tyneside, South Tyneside, Sunderland)      |
+|                | Our case study is Tyne and Wear county,          |
+|                | comprising the following local authorities:      |
+|                | Gateshead, Newcastle upon Tyne, North Tyneside,  |
+|                | South Tyneside, and Sunderland.                  |
 +----------------+--------------------------------------------------+
-| **How**        | Creating key **indicators** for assessing a      |
+| **How**        | Creating key indicators for assessing a          |
 |                | *baseline* scenario and *future* scenarios       |
 +----------------+--------------------------------------------------+
 | **Duration**   | 6 months                                         |
 +----------------+--------------------------------------------------+
 
```

Comment:
Is this outdated in light of more funding?

I think instead of writing the time ('6 months') here we should write the actual period that this project has been worked on. e.g. 'September 2022 to present'

Reply:
OK with the actual dates for me

Reply:
Fine with me